### PR TITLE
Live Routes Added

### PR DESCRIPTION
**Pull request by sfawcett123**
Live Routes added

Routes can be added now, as DNS has to be switched on the 8th April, until then they are not used.

### CODE REVIEW

File: terraform/paas/route.tf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 data "cloudfoundry_route" "app_route_internet" {
-  count    = var.additional_routes
   domain   = data.cloudfoundry_domain.internet.id
-  hostname = var.paas_additional_route_name
+  count    = length(var.paas_additional_route_names)
```
Comment (saliceti) [inline terraform/paas/route.tf]: It would be cleaner with a for_each here?

File: terraform/paas/route.tf [hunk 2]
```diff
@@ -1,7 +1,7 @@
 data "cloudfoundry_route" "app_route_internet" {
-  count    = var.additional_routes
   domain   = data.cloudfoundry_domain.internet.id
-  hostname = var.paas_additional_route_name
+  for_each = toset(var.paas_additional_route_names)
```
Comment (saliceti) [inline terraform/paas/route.tf]: Nice
By convention the `for_each` line comes first

File: terraform/paas/production.env.tfvars [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-paas_redis_1_name          = "get-into-teaching-prod-redis-svc"
-paas_additional_route_name = "beta-getintoteaching"
-instances                  = 2
-azure_key_vault            = "s146p01-kv"
-azure_resource_group       = "s146p01-rg"
+paas_space                  = "get-into-teaching-production"
+paas_app_route_name         = "get-into-teaching-app-prod"
+paas_app_application_name   = "get-into-teaching-app-prod"
+paas_redis_1_name           = "get-into-teaching-prod-redis-svc"
+paas_additional_route_names = ["beta-getintoteaching", "getintoteaching"]
```
Comment (saliceti) [inline terraform/paas/production.env.tfvars]: Misaligned
Needs `terraform fmt`?

Comment (saliceti) [inline terraform/paas/production.env.tfvars]: `paas_additional_route_names` is a bit misleading. Should it be something like `internet_hostnames`?

File: terraform/paas/route.tf (hunk 2 above)
Comment (sfawcett123) [reply terraform/paas/route.tf]: done

File: terraform/paas/production.env.tfvars (hunk 3 above)
Comment (sfawcett123) [reply terraform/paas/production.env.tfvars]: cannot see misalignment, did a terraform fmt, it didnt either

Comment (sfawcett123) [reply terraform/paas/production.env.tfvars]: changed to paas_internet_hostnames

**Review by saliceti — APPROVED** [review]

**Pull request merged**